### PR TITLE
Text updates

### PR DESCRIPTION
**Pull request by mjfrey**
Text edits for basket page.

### CODE REVIEW

**Comment by mjfrey** [comment]
@marjev @vkaracic

**Comment by vkaracic** [comment]
LGTM :+1:

File: ecommerce/templates/oscar/basket/partials/basket_content.html [hunk 1]
```diff
@@ -121,7 +121,7 @@
             <span><i class="fa fa-exclamation-circle"></i></span>
             {# Translators: <b></b> tags will bold the text within. Keep the tags and translate the text within. #}
             {% for line in lines %}
-                {% if line.certificate_type == 'Professional' or line.certificate_type == 'Verified' %}
+                {% if line.product.attr.id_verification_required or line.certificate_type == 'Verified' %}
```
Comment (clintonb) [inline ecommerce/templates/oscar/basket/partials/basket_content.html]: The second clause is not necessary.

**Comment by marjev** [comment]
LGTM :+1:

File: ecommerce/templates/oscar/basket/partials/basket_content.html (hunk 1 above)
Comment (mjfrey) [inline ecommerce/templates/oscar/basket/partials/basket_content.html]: @clintonb ah ok, so that is set for Verified as well?

Comment (clintonb) [inline ecommerce/templates/oscar/basket/partials/basket_content.html]: Yes.

**Pull request merged**